### PR TITLE
add a basic ucamwebauth configuration

### LINKED ISSUE #5: Wrong Makefile (includes references to other project)
https://github.com/uisautomation/sms-webapp/blob/d63400151b0d22bf552d7c1e795e1acff4238d38/Makefile#L19

### PR DESCRIPTION
**Pull request by rjw57**
Along with the configuration, we test that the admin actually requires login, that logging in via tests work with the newer force_login() method on the test client and we remove the confusing old GCloud stuff.

Closes #5

### CODE REVIEW

**Comment by abrahammartin** [comment]
CircleCI didn't execute tests for this branch, does this branch requires merging master into it first?

**Comment by rjw57** [comment]
No, this branch was created before I added my fork to CircleCI. Not sure if I can force a build...

**Comment by rjw57** [comment]
OK, deleting the branch and re-pushing appears to have kicked off a build: https://circleci.com/gh/rjw57/sms-webapp/6

File: smswebapp/test/test_admin.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+class AdminTests(TestCase):
+    def setUp(self):
+        # create superuser
+        self.superuser = User.objects.create_superuser(
+            'test0001', 'test@example.com', '')
+
+    def test_unauthenticated(self):
+        """Unauthenticated log in to admin redirects to login."""
+        r = self.client.get(reverse('admin:index'))
+        self.assertEqual(r.status_code, 302)
```
Comment (abrahammartin) [inline smswebapp/test/test_admin.py]: use assertRedirects?

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Requirements for the webapp itself
 django>=2.0
 psycopg2
+https://github.com/abrahammartin/django-ucamwebauth/archive/1.4.5.tar.gz#egg=django-ucamwebauth
```
[the remark this answers is not in the document]
Comment (rjw57) [reply requirements.txt]: Super - that wasn't true last night :).

File: smswebapp/test/test_admin.py (hunk 1 above)
Comment (rjw57) [reply smswebapp/test/test_admin.py]: Can we do the urlparse trick with assertRedirects? I don't really want to have to hard-code the ``?next=`` behaviour.

Comment (rjw57) [reply smswebapp/test/test_admin.py]: So, implemented in 4fac92e but it somehow feels like I'm hard-coding behaviour from the admin?

**Pull request merged**